src/lib/utils/series-utils.js

```diff
@@ -65,10 +65,21 @@ function collectSeriesTypesInfo(children) {
   return result;
 }
 
+/**
+ * Collect the stacked data for all children in use. If the children don't have
+ * the data (e.g. the child is invalid series or something else), then the child
+ * is skipped.
+ * Each next value of attr is equal to the previous value plus the difference
+ * between attr0 and attr.
+ * @param {Array} children Array of children.
+ * @param {string} attr Attribute.
+ * @returns {Array} New array of children for the series.
+ */
 export function getStackedData(children, attr) {
   const childData = [];
   let prevIndex = -1;
   children.forEach((child, childIndex) => {
```

Comment:
This looks like it can be re-written as a `const childData = children.map(…` to improve clarity and readability.